src/domUpdates.js

```diff
@@ -261,6 +277,12 @@ function renderFilteredRecipes() {
       <div class = "recipe-info">
         <p>${recipe.name}</p>
       </div>`)
+    if (!filtered.length) {
+      allRecipeDisplay.innerHTML = `
+      <div class="no-recipe-found-message">
+        <p>Sorry, ${currentUser.name}, no recipes match these tags.</p>
+      </div>`
+    }
```

Comment:
Nice job adding error handling here for when there are no recipes that include the checked tags!